pydis_site/apps/main/views/home.py

```diff
@@ -0,0 +1,106 @@
+import requests
+from django.conf import settings
+from django.shortcuts import render
+from django.utils import timezone
+from django.views import View
+
+from pydis_site.apps.main.models import RepoData
+
+GITHUB_API = "https://api.github.com/users/python-discord/repos"
+
+
+class Home(View):
+
+    def _get_api_data(self):
+        """Call the GitHub API and get information about our repos."""
+        repo_dict = {repo_name: {} for repo_name in settings.HOMEPAGE_REPOS}
+
+        # Fetch the data from the GitHub API
+        api_data = requests.get(GITHUB_API)
+        api_data = api_data.json()
+
+        # Process the API data into our dict
+        print(f"repo_dict = {repo_dict}")
+        for repo in api_data:
+            full_name = repo["full_name"]
+
+            if full_name in settings.HOMEPAGE_REPOS:
+                repo_dict[full_name] = {
+                    "full_name": repo["full_name"],
+                    "description": repo["description"],
+                    "language": repo["language"],
+                    "forks_count": repo["forks_count"],
+                    "stargazers_count": repo["stargazers_count"],
+                }
+        print(f"repo_dict after processing = {repo_dict}")
```

Comment:
I'm gonna assume these are debug prints and will hop out after the draft but leaving a comment here anyways

Reply:
yep yep, I've already removed this, I'll push new commits soon.